### PR TITLE
Feature/GitHub users

### PR DESCRIPTION
**Pull request by tonybaloney**
Add new actions 
* Get user
* list teams for organization
* Add (and invite if not member) user to team within organization

Bug exists in the current version of PyGitHub, so using their trunk.

### CODE REVIEW

**Comment by Kami** [comment]
Great, thanks.

And yeah, last time I checked, PyGithub was less than ideal. In fact, all of the Python libraries for interacting with Github I checked were less than ideal (kinda sad that Github doesn't develop an official one, anyways).

File: packs/github/actions/add_team_membership.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+name: add_team_membership
```
Comment (Kami) [inline packs/github/actions/add_team_membership.yaml]: add_team_member perhaps?

Anyway, not a big deal, I'm also fine with the current name :)

File: packs/github/actions/add_team_membership.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+parameters:
+  user:
+    type: "string"
+    description: "User to invite"
+    required: true
+  organization:
+    type: "string"
+    description: "Organization the team belongs to"
+    required: true
+  team:
```
Comment (Kami) [inline packs/github/actions/add_team_membership.yaml]: Probably better to be explicit and call it `team_id`, but would need to check existing actions. Maybe they don't use `id` suffix and current approach is better for consistency,

File: packs/github/actions/add_team_membership.yaml (hunk 1 above)
Comment (tonybaloney) [inline packs/github/actions/add_team_membership.yaml]: add_team_member is an existing action, which is being deprecated in the GitHub API. Depends on whether the user knows the github API!

**Comment by manasdk** [comment]
+1

Merging.

**Comment by manasdk** [comment]
@tonybaloney Appreciate the contribution!

**Pull request merged**